### PR TITLE
Garrison barracks overhaul + armory keys

### PR DESCRIPTION
**Pull request by feliniddotexe**
## About The Pull Request

TLDR; touches up the garrison building and gatehouse a bit, makes the keep and the garrison building a bit harder to break into, gives the armory a separate keyset from the rest of the garrison building.

<details>

<summary>Screenshots:</summary>

Downstairs:
image

Upstairs:
image

</details>

<details>

<summary>Full list of changes:</summary>

- Reworks the downstairs area of the garrison building into a common room with tables for eating/drinking and hanging out. 
- Replaces the windows in the garrison building with reinforced windows, the same kind the merchant & steward's buildings get.
- Moves the extra manor/garrison keys into locked closets/chests.
- The squire room actually has four beds now - for four squires.
- Moves some things around upstairs. All of the food and random cooking equipment that was previously laden around the barracks is now confined to one little miniature-kitchen-space.
- The gatehouse is a bit less cramped and a bit harder to break into.
- The garrison now has significantly better roof access; with several watchtowers leading up onto the roofs of the keep for those without climbing skills.
- Reduces the amount of easy rooftop routes to break into the keep that require no setup whatsoever from 6 to 3.

This PR also adds a separate keyset for the armory.
- The duke, the hand, the marshal, the veteran, the captain, the dungeoneer, the royal guards, the men at arms, the squires, and the wardens all have armory access.
- The consort, the two heirs, councilors, the court physician, and the servants all have basic garrison access, but cannot access the armory without asking someone to let them in.
- The guard captain and the steward both have spare armory keys in their office they can hand out if they want to.

</details>

## Why It's Good For The Game

The garrison building and the keep in general are comically easy to break into; this tightens up security just a bit and also makes it harder for other non-garrison keep roles like servants, councilors, and court physicians to raid the armory by making the armory use a separate key, while still allowing them entry to the garrison building itself for roleplay reasons such as tending to prisoners or cooking/cleaning.

### CODE REVIEW

**Comment by FreeStylaLT** [comment]
give it to the consort while you're at it por favor

**Comment by HeroWithYay** [comment]
+1

**Comment by Lutowski** [comment]
i would change lockid to most places that use garrison's to the keep instead & leave armory's lockid as is so maids can come into guards' quarters and maid all over the place

**Comment by YakumoChen** [comment]
Disagree with the keyset changes. I think it's fine to have extra access to the armoury, makes for interesting plots too if a butler or someone suddenly decides to go axe crazy. I think we can have a little leeway for thickening plots.

**Comment by FreeStylaLT** [comment]
I can't say I approve of squires losing armory access. It'll be a lot more cumbersome to experiment or learn on non-peak-pop rounds, where someone with an Armory key might not be easily available or present.
